testdata/resolvers/pipelineruns/bundles-resolver-pipelinerun-param.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: bundles-resolver-pipelinerun-param
+spec:
+  pipelineRef:
+    resolver: bundles
+    params:
+    - name: bundle
+      value: quay.io/openshift-pipeline/resolver-test-bundles:1.0
```

Comment:
We should document somewhere what this bundle contains and how it was created. If it's short, even a comment here would do the job, if it's longer, we should find a better place.